lib/diagnostic.rb

```diff
@@ -105,15 +125,21 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+
+class Lion < Cat
+  include Carnivorous
+  def roar
+    puts 'ROAR!'
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+# composition allows you to get information from multiple modules.
```

Comment:
Yes, but this doesn't tell us about the advantages that it has over inheritance.  One advantage over inheritance is that with inheritance, classes can only inherit from one other class.